src/ds/d500/d500-device.cpp

```diff
@@ -719,17 +719,27 @@ namespace librealsense
 
     bool d500_device::check_symmetrization_enabled() const
     {
-        using namespace ds;
-        command cmd(GET_HKR_CONFIG_TABLE,
-            static_cast<int>(d500_calib_location::d500_calib_ram_memory),
-            static_cast<int>(d500_calibration_table_id::stream_pipe_config_id),
-            static_cast<int>(d500_calib_type::d500_calib_dynamic));
-        auto res = _hw_monitor->send(cmd);
+        try
```

Comment:
Can you add a comment in the code please about why we have this fallback flow?

Reply:
done